public_html/wp-content/themes/wporg-events-2023/postcss/blocks/wporg-google-map.pcss

```
@@ -66,8 +66,25 @@
 		}
 	}
 
-	.wporg-google-map__date-time-separator {
-		@media (--medium) {
+	.wporg-google-map__date {
+		position: relative;
+	}
+
+	.wporg-google-map__date:after {
+		content: '';
+		margin-top: -1px; /* vertical-middle doesn't subtract the size of the element */
+		margin-left: 8px;
+		margin-right: 8px;
```

This looks a little tight to me, Figma suggests:

```suggestion
		margin-left: 10px;
		margin-right: 10px;
```